streamlit.py

```
@@ -10,42 +10,139 @@
 from pathlib import PosixPath
-from typing import cast
+from typing import cast, Any
+import json
 
-import streamlit as st
 from anthropic import APIResponse
-from anthropic.types import (
-    TextBlock,
-)
-from anthropic.types.beta import BetaMessage, BetaTextBlock, BetaToolUseBlock
+from anthropic.types import Message
```

Ruff (0.8.0)

`anthropic.types.Message` imported but unused

Remove unused import: `anthropic.types.Message`

(F401)